frontend/js/src/common/brainzplayer/SpotifyPlayer.tsx

```diff
@@ -436,6 +444,9 @@ export default class SpotifyPlayer
           const userToken = await refreshSpotifyToken();
           this.accessToken = userToken;
           this.authenticationRetries = 0;
+          if (spotifyUser) {
+            spotifyUser.access_token = userToken;
+          }
```

Comment:
Do we need an else condition here, where we populate the `spotifyAuth` object if it doesn't exist, with the new access token ?
I guess we won't have the `permissions` though